KUBEVIRTCI_LOCAL_TESTING.md

```diff
@@ -65,6 +65,18 @@ export KUBEVIRT_DEPLOY_PROMETHEUS_ALERTMANAGER=true
 export KUBEVIRT_DEPLOY_GRAFANA=true
 ```
 
+#### start cluster with Kubernetes feature gates
+To enable/disable Kubernetes feature gates, export the following variable before running `make cluster-up`:
+
+```bash
+export K8S_FEATURE_GATES="<FeatureGate1>=true,<FeatureGate2>=false"
+```
+
+for feature gates that require runtime config the following variable should be exported:
+```bash
+export K8S_API_RUNTIME_CONFIG="<RuntimeConfig>=true"
```

Comment:
What does it mean, setting it to true?

Reply:
According to https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/ we can also disable runtime-config

Reply:
`--runtime-config <comma-separated 'key=value' pairs>`

Ah, got it.